### PR TITLE
Issue #1163 - Convert SunInfo to use templates

### PR DESCRIPTION
**Pull request by Mailkov**
IA Page: https://duck.co/ia/view/sun_info

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/sun_info/sun_info.css [hunk 1]
```diff
@@ -1,55 +1,55 @@
-.zci--answer .zci--suninfo {
+.zci--sun_info {
     font-size: 1.75em;
```
Comment (moollaza) [inline share/goodie/sun_info/sun_info.css]: Let's remove this and as much other CSS possible. It looks like it's causing the IA to have a ton of extra space around it.

We can use our CSS classes to handle font size and color.

File: share/goodie/sun_info/content.handlebars [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class='suninfo--header text--secondary'>
```
Comment (moollaza) [inline share/goodie/sun_info/content.handlebars]: Let's use the `c-base__sub` class. It will take care of the size and colour for the subtitle.

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

I'm going to add back a little CSS to normalize the sizing and I'm gonna move the title into a `title_content` template :+1:

**Pull request merged**